extension/pybindings/pybindings.cpp

```diff
@@ -168,13 +168,14 @@ class Module final {
   explicit Module(
       std::unique_ptr<DataLoader> loader,
       std::unique_ptr<ETDumpGen> tracer = nullptr,
-      size_t debug_buffer_size = 0)
+      size_t debug_buffer_size = 0,
+      Program::Verification load_program_config =
```

Comment:
"config" is vague; `load_program_verification` or just `program_verification` would be more descriptive and precise. Same for other params with the same name.